frontend/src/components/Forms/CoffeeChatsForm/CoffeeChatsForm.tsx

```diff
@@ -0,0 +1,193 @@
+            Select a team member: <span className={styles.red_color}>*</span>
+          </label>
+          <div className={styles.center_and_flex}>
+            {membersList ? (
+              <Dropdown
+                placeholder="Select a team member"
+                fluid
+                search={(options, query) =>
+                  options.filter((option) => option.key.toLowerCase().includes(query.toLowerCase()))
+                }
+                selection
+                value={member?.netid ?? ''}
+                text={member ? `${member.firstName} ${member.lastName} (${member.netid})` : ''}
+                options={membersList
+                  .sort((m1, m2) =>
+                    `${m1.firstName} ${m1.lastName}`.localeCompare(`${m2.firstName} ${m2.lastName}`)
+                  )
+                  .map((member) => ({
+                    key: member.netid,
+                    label: (
+                      <div className={styles.flex_start}>
+                        {member.firstName} {member.lastName} ({member.netid})
+                      </div>
+                    ),
+                    value: member.netid
+                  }))}
+                onChange={(_, data) => {
+                  setMember(membersList.find((member) => member.netid === data.value));
+                }}
+              />
+            ) : undefined}
+          </div>
+        </div>
+        <div className={styles.inline}>
+          <label className={styles.bold}>
+            Which category does this coffee chat fulfill?{' '}
+            <span className={styles.red_color}>*</span>
+          </label>
+          <div className={styles.center_and_flex}>
+            <Dropdown
+              placeholder="Select a category"
+              fluid
+              search={(options, query) =>
+                options.filter((option) => option.key.toLowerCase().includes(query.toLowerCase()))
+              }
+              selection
+              value={category}
+              text={category}
+              options={bingoBoard.flat().map((category, _) => ({
+                key: category,
+                label: <div className={styles.flex_start}>{category}</div>,
+                value: category
+              }))}
+              onChange={(_, data) => {
+                const foundCategory = bingoBoard.flat().find((category) => category === data.value);
+                setCategory(foundCategory || '');
+              }}
+            />
+          </div>
+        </div>
+        <div className={styles.inline}>
+          <label className={styles.bold}>
+            Submit a Slack link to your coffee chat! <span className={styles.red_color}>*</span>
+          </label>
+          <p className={styles.margin_bottom_zero}>
+            Go to your coffee chat image in the #coffee-chats channel, and click on 'Copy link to
+            file'. Example: https://cornelldti.slack.com/files/A05SULTP4V7/F06LF0ZNCQM/img_4661.jpg
```

Comment:
I'd like to encourage students to actually submit the slack post instead of the image link actually, since that has more info (has both people tagged and the sender of the slack message). What do you think?

Reply:
Yeah I agree!